### ISSUE TITLE
How can we run multiple splash instances on the same port without docker?

### ISSUE THREAD

**Issue by Sanjeetkumar163** [scrapinghub/splash#824]
I have installed splash manually and want to run multiple splash instances on the same port like we do this by using docker-compose. How can we do this?

**Comment by lopuhin** [comment]
@Sanjeetkumar163  https://github.com/TeamHG-Memex/aquarium should handle this.

**Comment by Sanjeetkumar163** [comment]
@lopuhin  Yes, it is there but I don't want to run this through docker/docker-compose rather want to install it [manual-way](https://splash.readthedocs.io/en/stable/install.html#ubuntu-16-04-manual-way).

**Comment by lopuhin** [comment]
@Sanjeetkumar163 ah I see, then I'm not aware of anything ready-made, but you can use aquarium as an example, borrowing haproxy config, and running haproxy and several splash instances in some other way, e.g. with supervisord.
